node/src/service.rs

```diff
@@ -641,10 +644,22 @@ async fn start_sh_services(
 
     let blockchain_service_handle = spawn_blockchain_service(&task_spawner).await;
 
-    let sh_handler = StorageHubHandler::new(
+    let file_storage = Arc::new(RwLock::new(InMemoryFileStorage::new()));
+    let forest_storage = Arc::new(RwLock::new(InMemoryForestStorage::new()));
```

Comment:
Remind me to have a discussion on tokio's `RwLock` using a *write-first* approach.